TeapotTests/TeapotTests.swift

```diff
@@ -78,6 +78,64 @@ class TeapotTests: XCTestCase {
         self.waitForExpectations(timeout: 20.0)
     }
 
+    func testPutWithJSONData() {
+        let expectation = self.expectation(description: "Put with JSON")
+
+        let dict = "{\"foo\":\"bar\"}"
+        guard let jsonData = dict.data(using: .utf8) else {
+            XCTFail("Couldn't incode dic string")
```

Comment:
This doesn't look right 😅

Reply:
I wasn't tired when I was doing this... 🤦‍♀️